src/NuGetGallery.Core/GitHub/IGitHubUsageConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NuGetGallery.GitHub
+{
+    public interface IGitHubUsageConfiguration
+    {
+        /// <summary>
+        /// Returns a NuGetPackageGitHubInformation object that contains the information about a NuGet package.
+        /// NOTE: If a packageId has no information, the NuGetPackageGitHubInformation's TotalRepos will be 0
+        /// and the Repos list will be empty
+        /// 
+        /// throws an ArgumentException if the packageId is null
+        /// </summary>
+        /// <param name="packageId">NuGet package id</param>
```

Comment:
I would remove line 10 and add that note right on the `param` itself:

```suggestion
        /// <param name="packageId">NuGet package id, cannot be null</param>
```